bin/build-module.js

```diff
@@ -30,20 +30,7 @@ function buildModule(filepath) {
   // All external modules are assumed to be CommonJS, and therefore should
   // be skipped by Rollup. We may revisit this later.
   var depsToSkip = Object.keys(topPkg.dependencies || {});
-  depsToSkip = depsToSkip.concat([
-    'crypto', 'fs', 'events', 'path'
-  ]);
-
-  if (pkg.name === 'pouchdb-for-coverage') {
-    // special case - for the coverage reports, the whole thing is
-    // bundled into one index.js. so we don't want to externalize
-    // the pouchdb repos
```

Comment:
Okay, `pouchdb-for-coverage/lib/index.js` and `pouchdb/lib/index.js` are both the same before and after this change, so LGTM.